app/src/pages/playground/playgroundUtils.ts

```diff
@@ -724,93 +720,6 @@ export const getToolName = (tool: Tool): string | null => {
   }
 };
 
-/**
- * Best effort attempts to convert instance tools to the providers schema
- * If the tool definition cannot be converted, it will be returned as is
- * @returns A list of playground {@link Tool|Tools}
- */
-export const convertInstanceToolsToProvider = ({
```

Comment:
moved into playground store utils so that we don't have circular imports still some imports coming from here up but probably okay since it's all types that get passed the other way

Reply:
From what I understand circular type imports are no problem

Reply:
yeah that's what i was figuring, wante dto move these cause it felt weird to be importing from page utils up to here, we still have a few coming up - they have types that are generated from relay though so they also feel like they should be down in /pages so i just left for now